### ISSUE TITLE
Update to node 18 and node 20

### ISSUE THREAD

**Issue by guillermodelagala** [NikolajDL/gcloud-sdk-nodejs#3]
Please, as node 16 is going to be deprecated, could you add compatibility with node 18 and node 20?

**Comment by NikolajDL** [comment]
Hello Guillermo,

I've restructured the image a bit, because merging two images, like I did previously, doesn't really work with the Docker buildkit anymore. 

Now, instead of merging two images, I use the Google Cloud SDK image as the base, and then install the specified node version instead. 
As part of this restructuring, I've also cleaned up the Docker hub repo a bit. I've completely removed the alpine tags, since they were based on the nodejs image being Alpine. Since I now simply install node in the image, the node Alpine version no longer makes sense. I may make gcloud alpine tags in the future, if requested. The removal of the Alpine tags may cause some disruption for those who relied on them previously. 

The end result should be the exact same, but the images ought be smaller and this way, I've been able to automate the build of the images. 

TLDR: I've updated the latest version to point to node 21, the lts version to point to 20 and then added 20 and 18 as separate versions.

Please let me know whether you experience any issues.

**Comment by guillermodelagala** [comment]
Thank you very much @NikolajDL for your work. I'll check the 18 version by now.
